README.md

```diff
@@ -49,6 +49,30 @@ a course about testing, each level is a separate `test-suite` in the
 
 Solutions are on the `solutions` branch, one commit per level.
 
+## Running with stack
+
+You will need to run a few additional commands when using [stack](https://docs.haskellstack.org/en/stable/README/):
+
+```bash
+
+# initialise
+$ stack init
+
+# add dependencies
+$ stack add hedgehog
+$ stack add tasty
+$ stack add tasty-hedgehog
+
+# run ghci
+stack ghci
```

Comment:
Some of the other stack-users said that `stack test --file-watch :level01` and `stack ghci :level01` worked. If this is true for your machine, can you please revise this? That way we won't have to ask users to run a bunch of package-management commands.

If you don't have time to revise this, can you please test the commands and let me know whether they ran or not? Then I can merge and update the docs.

Reply:
`stack test --file-watch :level01` worked a treat - updating now.